front/app/containers/IdeasNewPage/IdeasNewSurveyForm/index.tsx

```diff
@@ -242,10 +244,8 @@ const IdeasNewSurveyForm = ({ project }: Props) => {
               />
             </Box>
           </Box>
-        ) : inputSchemaError ? null : (
-          <FullPageSpinner />
```

Comment:
I will actually put back the full-page spinners I removed in variants of the idea form in a subsequent PR. I need to modify the hook to also pass isLoading and use that instead.